### PR TITLE
Fix test_change_client_ocs_version_and_stop_heartbeat test

### PR DESCRIPTION
**Pull request by fbalak**
* Fixes https://github.com/red-hat-storage/ocs-ci/issues/9394
* Fixes test_change_client_ocs_version_and_stop_heartbeat test case
* Updates check_alert_list function to correctly check alerts with same label but different message

### CODE REVIEW

File: tests/functional/monitoring/conftest.py [hunk 1]
```diff
@@ -1177,7 +1177,7 @@ def change_client_version():
         nonlocal client
         nonlocal original_cluster
         # run_time of operation
-        run_time = 60 * 3
+        run_time = 60 * 7
```
Comment (nehaberry) [inline tests/functional/monitoring/conftest.py:1180]: @fbalak why is this value set at 60*7 ? Could you please clarify to me?

**Review by nehaberry — APPROVED** [review]
LGTM.

**Review by dahorak — APPROVED** [review]
LGTM

File: tests/functional/monitoring/conftest.py (hunk 1 above)
Comment (DanielOsypenko) [reply tests/functional/monitoring/conftest.py:1180]: StorageClientHeartbeatMissed - description has 120(s) for Warning and 300(s) for Critical.

Expressions in Prometheus rules confirming. Waring expression for reference.
`[(time() - 120) > (ocs_storage_client_last_heartbeat > 0)](https://console-openshift-console.apps.ibm-baremetal1.qe.rh-ocs.com/monitoring/query-browser?query0=(time()%20-%20120)%20%3E%20(ocs_storage_client_last_heartbeat%20%3E%200))`

image
image

StorageClientIncompatibleOperatorVersion immediate as far as I understand. No interval in the description. 

So all together should be 420sec enough, which is the same as Filip set here

**Review by DanielOsypenko — APPROVED** [review]

**Review by dahorak — APPROVED** [review]

**Review by ebondare — APPROVED** [review]

**Review by DanielOsypenko — APPROVED** [review]

**Pull request merged**